error: reinterpret_cast from 'nullptr_t' to 'OpenBabel::transform3d *' is not allowed

Index: src/math/spacegroup.cpp
--- src/math/spacegroup.cpp.orig
+++ src/math/spacegroup.cpp
@@ -280,14 +280,14 @@ namespace OpenBabel
   transform3d const * SpaceGroup::BeginTransform(transform3dIterator &i) const
   {
     i = m_transforms.begin ();
-    return (i == m_transforms.end())? reinterpret_cast<transform3d*>(NULL): *i++;
+    return (i == m_transforms.end())? reinterpret_cast<transform3d*>(0): *i++;
   }
 
   /*! 
    */
   transform3d const * SpaceGroup::NextTransform(transform3dIterator &i) const
   {
-    return (i == m_transforms.end())? reinterpret_cast<transform3d*>(NULL): *i++;
+    return (i == m_transforms.end())? reinterpret_cast<transform3d*>(0): *i++;
   }
 
   /*! 
